tags/module.md

```diff
@@ -17,6 +17,9 @@ It may optionally be used to rename a module whose name TypeDoc guesses incorrec
 The TSDoc specified [`@packageDocumentation`](/tags/packageDocumentation) tag can also be used to mark
 a comment as referring to the file, but cannot be used to rename the module.
 
+**NOTE:** A comment block where the `@module` tag is used must be the first comment on the page. So it is
```

Comment:
"On the page" is somewhat confusing as it's not on a page yet, it's in a source file

Reply:
agreed